draft-ietf-tls-tls13.md

```diff
@@ -4936,9 +4959,11 @@ Where Secret is either the early_exporter_master_secret or the
 exporter_master_secret.  Implementations MUST use the exporter_master_secret unless
 explicitly specified by the application. The early_exporter_master_secret is
 defined for use in settings where an exporter is needed for 0-RTT data.
-A separate interface for the early exporter is RECOMMENDED, especially
-on a server where a single interface can make the early exporter
-inaccessible.
+A separate interface for the early exporter is RECOMMENDED.  This is especially
+relevant on the server, where an implementation that processes the ClientHello
+and generates its first flight immediately would not be able to produce
+early exporter values because the key schedule has already computed the
+handshake secret and discarded the early secret.
```

Comment:
The actual rationale was for security, so I would prefer not to add this text. There are lots of ways to screw up your database.

Reply:
So you want to go back to "especially on a server where a single interface can make the early exporter inaccessible", or just stop after "is RECOMMENDED."?